cirq-core/cirq/transformers/analytical_decompositions/two_qubit_state_preparation.py

```diff
@@ -106,3 +106,37 @@ def prepare_two_qubit_state_using_cz(
     return op_list + _1q_matrices_to_ops(
         np.dot(u, np.linalg.inv(u_CZ)), np.dot(vh.T, np.linalg.inv(vh_CZ.T)), q0, q1
     )
+
+
+def prepare_two_qubit_state_using_iswap(
+    q0: 'cirq.Qid', q1: 'cirq.Qid', state: 'cirq.STATE_VECTOR_LIKE'
```

Comment:
Can we also add an option to prepare using `ISWAP_INV`, similar to the `prepare_two_qubit_state_using_sqrt_iswap` ? Or is that non-trivial / not needed here?